lib/src/test/java/com/auth0/jwt/algorithms/HMACAlgorithmTest.java

```diff
@@ -64,8 +64,6 @@ public void shouldPassHMAC256Verification() throws Exception {
         Algorithm algorithmBytes = Algorithm.HMAC256("secret".getBytes(StandardCharsets.UTF_8));
         JWT jwt = JWT.require(algorithmString).withIssuer("auth0").build();
         DecodedJWT decoded = jwt.decode(token);
```

Comment:
is `decode` verifying the token signature now?? why you no longer verify the `Algorithm#verify(bytes)` signature?? (algorithmBytes)

Reply:
decode has been verifying token signature for a while now